scripts/add-gpgme-and-gnupg21-to-venv

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+GNUPG_DEPS="\
+    https://gnupg.org/ftp/gcrypt/npth/npth-1.3.tar.bz2 \
+    https://gnupg.org/ftp/gcrypt/libksba/libksba-1.3.5.tar.bz2 \
+    https://gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.25.tar.bz2 \
+    https://gnupg.org/ftp/gcrypt/libassuan/libassuan-2.4.3.tar.bz2 \
+    https://gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.7.5.tar.bz2 \
+    https://gnupg.org/ftp/gcrypt/gnupg/gnupg-2.1.17.tar.bz2 \
+    https://gnupg.org/ftp/gcrypt/gpgme/gpgme-1.8.0.tar.bz2"
+
+[ "$VIRTUAL_ENV" == "" ] && {
+    echo "Please activate your virtualenv first!"
+    exit 1
+}
+echo
+echo "About to download, build and install GPGME 1.8.0 and GnuPG 2.1.17"
+echo "into: $VIRTUAL_ENV"
+echo
+echo "Press ENTER to continue."""
+read
+
+set -x
+set -e
+
+mkdir -p "$VIRTUAL_ENV/gnupg-build"
+cd "$VIRTUAL_ENV/gnupg-build"
+
+for package in $GNUPG_DEPS; do
+    tarball=$(basename $package)
+    srcdir=$(basename $package |sed -e s/.tar.bz2//)
+
+    if [ ! -e "$tarball" ]; then
+        rm -f dl.tmp
+        wget "$package" -O dl.tmp
```

Comment:
The links are all https links, so I'm not entirely sure what problems we'd be solving?